CAM Platform/workspaces.md

```diff
@@ -13,812 +13,892 @@ Manage and perform actions on workspaces.
 
 | Resource |Description |
 |----------|------------|
-| GET /services/workspaces | Gets the list of workspaces. |
-| POST /services/workspace | Creates a new team workspace.   |
-
+| [GET /services/workspaces](#get-servicesworkspaces) | Gets the list of workspaces |
+| [POST /services/workspace](#post-servicesworkspace) | Creates a new team workspace |
 
 **Perform Other Workspace Operations**
 
 | Parameter | Description |
 |-----------|-------------|
-| GET /services/workspaces/{workspace_id} | Fetches an existing workspace.|
-| PUT /services/workspaces/{workspace_id}| Updates an existing workspace.|
-| DELETE /services/workspaces/{workspace_id}| Deletes an existing workspace.|
-| GET /services/workspaces/{workspace_id}/providers | Gets the cloud providers registered in a workspace. |
-| GET /services/workspaces/{workspace_id}/boxes | Gets the boxes in a workspace. |
-| GET /services/workspaces/{workspace_id}/instances | Gets the instances in a workspace. |
-
-### GET /services/workspaces
-
-Gets a list of all accessible workspaces. There are two types of workspaces: personal workspaces for a single user and team workspaces that can have many members and organizations.
-
-**Normal Response Codes**
+| [GET /services/workspaces/{workspace_id}](#get-servicesworkspacesworkspace_id) | Fetches an existing workspace |
+| [PUT /services/workspaces/{workspace_id}](#put-servicesworkspacesworkspace_id) | Updates an existing workspace |
+| [DELETE /services/workspaces/{workspace_id}](#delete-servicesworkspacesworkspace_id) | Deletes an existing workspace |
+| [GET /services/workspaces/{workspace_id}/providers](#get-servicesworkspacesworkspace_idproviders) | Gets the cloud providers registered in a workspace |
+| [GET /services/workspaces/{workspace_id}/boxes](#get-servicesworkspacesworkspace_idboxes) | Gets the boxes in a workspace |
+| [GET /services/workspaces/{workspace_id}/instances](#get-servicesworkspacesworkspace_idinstances) | Gets the instances in a workspace |
 
-* 200
 
-**Error Response Codes**
+## GET /services/workspaces
+Gets the default workspace if no parameter is provided. If ids parameter (a list of workspace ids) is provided then it fetches all the workspaces corresponding to that list.
```

Comment:
This actually retrieves the default workspace only if the users has no other privilege. If no `ids` is provided, it returns a list of workspaces the user has access to. In any case, the output is an array. Please check.